TRADEMARKS.md

```diff
@@ -0,0 +1,231 @@
+
+You may use the Word Marks, but not the Logos, to truthfully describe the origin of the software that you are providing, that is, that the code you are distributing is a modification of our software. You may say, for example, that "this software is derived from the source code for AmpliPi software", or "this disk image contains AmpliPi software".
+
+Of course, you can place your own trademarks or logos on versions of the software to which you have made substantive modifications, because by modifying the software you have become the origin of that exact version. In that case, you should not use our Logos.
+
+In any event, you must ensure that you are in compliance with the software license for AmpliPi, which requires (among other things) that any modifications you make to AmpliPi must likewise be made open source under the same license. See accompanying software license file for more information.
+
+### Statements about compatibility, interoperability or derivation
+
+You may use the Word Marks, but not the Logos, to truthfully describe the relationship between your software and ours. Our Mark should be used after a verb or preposition that describes the relationship between your software and ours. So you may say, for example, "Bob's software for the AmpliPi platform" but may not say "Bob's AmpliPi software." Some other examples that may work for you are:
+
+- [Your software] works with AmpliPi software
+- [Your software] uses AmpliPi software
+- [Your software] is compatible with AmpliPi software
+- [Your software] is powered by AmpliPi software
+- [Your software] runs on AmpliPi software
+- [Your software] for use with AmpliPi software
+- [Your software] for AmpliPi software
+
+### Use of trademarks to show community affiliation
+
+This section discusses the use of our Marks for software such an application themes, skins and personas. The use of our Marks on websites is discussed below.
+
+You may use the Word Marks and the Logos in themes, personas, or skins for applications to show your support for the Project, provided that the use is non-commercial and the use is clearly decorative, as contrasted with a use that appears to be the branding for a website or application. 
+
+## Uses for which we may grant a license
+
+### Distribution of hardware
+
+This kind of use concerns hardware associated with AmpliPi (This may include: hardware that comes preloaded with AmpliPi software, hardware that is advertised, marketed, or documented as being compatible with AmpliPi software, or hardware otherwise intended for use with AmpliPi software.)
+
+You may use use the Word Marks and the Logos in association with hardware devices if and only if ALL of the following conditions are met:
+- The AmpliPi software installed on the device is only be modified so far as necessary to operate on the hardware platforms and the essential functions of the software are unchanged. 
```

Comment:
I think "The AmpliPi software installed on the device is only to be modified so far as necessary to operate on the hardware platforms, while the essential functions of the software are unchanged." or something similar sounds better here. The last chunk of the sentence sounds strange to me